packages/terra-application/src/application-error-boundary/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './ApplicationErrorBoundary';
```

Comment:
It allows consumers to import the ApplicationErrorBoundary like this:
`import ApplicationErrorBoundary from 'terra-application/lib/application-error-boundary';`

instead of

`import ApplicationErrorBoundary from 'terra-application/lib/application-error-boundary/ApplicationErrorBoundary';`

The other sub-directories have index files as well, but they're exporting multiple components. This one is a little weird since we only have a single component, but I put it in here to be consistent.

It might make more sense to just export everything from a single index file at the application root. I was trying to logically divide things a bit, but everything here is likely going to be included in the bundle anyway.

Reply:
I personally would prefer a root index export so you could do either:
```
import TerraApplication from 'terra-application';
// or 
import {  ApplicationErrorBoundary } from 'terra-application';
```